examples/basic-jsx-precompile/index.html

```diff
@@ -11,23 +11,25 @@
       <p>
         If you can see this, React is not running. Try running:
       </p>
-      <pre>npm install -g react-tools
```

Comment:
Nothing I just wanted prettier HTML. Comparing styling now though, looks like leaving as purely `pre` will be better.